### DISCUSSION TITLE
How to create two ceph clusters with mirroring enabled in both cephFS

### DISCUSSION THREAD

**Discussion by ahmedTouati** [rook/rook#7966] [Q&A] [answered]
we're using rook-operator v1.5.8, and we were looking for a Disaster recovery strategy to implement in rook-ceph, so we thought of creating two different rook-ceph clusters in the same kubernetes cluster and enable mirroring since it's available in v1.6.0, then implement mirroring in two regions separated rook-ceph clusters, as if we have an active and standby rook-ceph clusters
is that the correct way to do it (using cephFS mirroring)?

**Comment by Madhu-1** [comment, marked as answer]
CephFS mirroring is not fully completed yet, its ongoing work at https://github.com/rook/rook/pull/7604

**Reply by ahmedTouati** [reply]
@Madhu-1 Is there any Disaster Recovery Strategy we can implement in Rook like creating Snapshots of the disks than use them in another ceph cluster?

**Reply by Madhu-1** [reply]
Nothing am aware of except mirroring. @leseb any idea?

**Reply by leseb** [reply]
It depends on the storage layer:

* for Object at-rest: use object store multisite
* for Block use RBD Mirroring
* for File wait for CephFS mirroring to be ready in Rook or go with periodic rsync.

**Reply by ahmedTouati** [reply]
@leseb if i follow the steps here wouldn't that help ?
https://docs.ceph.com/en/latest/dev/cephfs-mirroring/

**Reply by leseb** [reply]
Yes and No, we need Ceph Pacific 16.2.5 for cephfs-mirror to work correctly in Rook. So be patient.

**Reply by MurzNN** [reply]
Is Rook Ceph mirroring possible to configure between separate Kubernetes clusters? Because it's much better for disaster recovery if the main K8s cluster totally dies, the second should rescue the data!
